src/utils.js

```diff
@@ -0,0 +1,16 @@
+// @flow
```

Comment:
If we are modularising the storage directory structure, why not modularise utils too? `/utils/serialize.js`